core/chains/evm/logpoller/orm.go

```diff
@@ -63,7 +63,7 @@ func (o *ORM) InsertFilter(filter Filter, qopts ...pg.QOpt) (err error) {
 		filter.Name, utils.NewBig(o.chainID), addresses, events)
 }
 
-// DeleteFilter removes all events,address pairs associated with the filter
+// DeleteFilter removes all events,address pairs associated with the Filter
```

Comment:
I'm confused about these capitalization changes.  Why did this and a few others change from lowercase to uppercase, but most of the others changed from upper to lower?